python/mxnet/ndarray/numpy/_op.py

```diff
@@ -1933,7 +1933,7 @@ def _unary_func_helper(x, fn_array, fn_scalar, out=None, **kwargs):
     if isinstance(x, numeric_types):
         return fn_scalar(x, **kwargs)
     elif isinstance(x, NDArray):
-        return fn_array(x, out=out, **kwargs)
```

Comment:
Since legacy operators make heavy use of `_unary_func_helper`, what about leave it as it was? We may have a new helper function for `_api_internal`, which may support `*args` instead of `**kwargs`

Reply:
In fact, these simple unary ops do not support any `kwargs` in MXNet, `kwargs` only need to make scalar's result right and has uniform error handling by using `@wrap_np_unary_func`. So, move `**kwargs` here is a very simple solution.

Reply:
My concern is that in the future, people may want to add other unary operators with kwargs (like tile) with legacy ffi. They may not notice that `**kwargs` are no longer supported by `_unary_func_helper`.